app/src/main/java/com/example/trainingintive/dog_images_feature/domain/model/DogImageUrl.kt

```diff
@@ -1,12 +1,9 @@
 package com.example.trainingintive.dog_images_feature.domain.model
 
-import androidx.room.Entity
-import androidx.room.PrimaryKey
-import com.squareup.moshi.Json
+import com.example.trainingintive.dog_images_feature.data.entity.DogImageUrlDb
 
-@Entity
 data class DogImageUrl(
-    @PrimaryKey
-    @Json(name = "message")
     val url: String
-)
+) {
+    fun toDatabaseEntity(): DogImageUrlDb = DogImageUrlDb(url)
```

Comment:
Mapper from domain to data layer should not be in domain layer (remember – domain don't know anything about data layer). Move mapper to data layer for e.g. to DogImageUrlDb as a extension to DogImageUrl